win32-setup/Makefile

```diff
@@ -29,22 +29,40 @@ curl:
 	cd curl-7.66.0; ./configure --host=i686-w64-mingw32 --target=i686-w64-mingw32 CC=$(CC) CXX=$(CXX) CFLAGS="$(CFLAGS)" LIBS=$(LIBS) --disable-shared --without-zlib --without-ssl --with-mbedtls=$(PWD) --enable-optimize --enable-http --disable-ftp --disable-file --disable-ldap  --disable-ldaps  --disable-rtsp --enable-proxy --disable-dict --disable-telnet --disable-tftp --disable-pop3 --disable-imap --disable-smb --disable-smtp --disable-gopher --enable-manual
 	$(MAKE) -j4 -C curl-7.66.0
 
-lokinet-bootstrap.exe: mbedtls curl
+lokinet-bootstrap.exe: mbedtls curl dbghelp
 	cp curl-7.66.0/src/curl.exe $@
 	wget -O rootcerts.pem https://curl.haxx.se/ca/cacert.pem
 	cp ../LICENSE .;unix2dos LICENSE LICENSE
-else
 
-all: regdbhelper.dll lokinet-bootstrap.exe
+dbghelp:
+	wget https://snowlight.net/loki/win32-dist/dbghelp32.dll
+	wget https://snowlight.net/loki/win32-dist/dbghelp64.dll
 
+else
 regdbhelper.dll:
-	scp despair@10.10.10.6:loki-network/win32-setup/*.dll .
+	scp despair@10.10.10.3:web/loki/win32-dist/*.dll .
 
 lokinet-bootstrap.exe:
-	scp despair@10.10.10.6:loki-network/win32-setup/*.exe .
+	scp despair@10.10.10.3:web/loki/win32-dist/*.exe .
```

Comment:
this should be buildable for everyone not just maintainers.

Reply:
yeah that was left over when i moved the prebuilt assets to a TLS server